tests/fiaas_deploy_daemon/deployer/kubernetes/deployment/test_datadog.py

```diff
@@ -43,7 +43,8 @@ def test_noop_when_not_enabled(self, datadog, app_spec, deployment):
 
     @pytest.mark.parametrize("best_effort_required", (False, True))
     def test_adds_env_when_enabled(self, datadog, app_spec, deployment, best_effort_required):
-        app_spec = app_spec._replace(datadog=True)
+        datadog_spec = app_spec.datadog._replace(enabled=True, tags=dict())
```

Comment:
Should this be tags = None?

Reply:
I don't think. Empty dict should be fine as default, as in https://github.com/fiaas/fiaas-deploy-daemon/pull/11/files#diff-13d0cf28b75b00450ea41240032087a4R31 you're assuming that tags is a dict (with None that line would fail).

Reply:
Yeah, I was wondering if we should actually support None at datadog.py (e.g. for backwards compatibility reasons with v2).  Thanks!